src/components/tasks/CreateButton.jsx

```diff
@@ -0,0 +1,22 @@
+import PropTypes from "prop-types";
+import "src/components/tasks/CreateButton.scss";
+import { ADD_ICON } from "src/utils/constants/constants";
+
+const CreateButton = ({ showTask, onShow }) => {
```

Comment:
We have wrote a `ButtonCreator` component before ? Can we reuse that instead of initializing another similar component here ?